### PR TITLE
made vibrators usable without batteries

### PR DESCRIPTION
**Pull request by Tsunder**
20% more exhausting and 25% slower morale increase rate without batteries

### CODE REVIEW

**Comment by codemime** [comment]
I'd nerf them heavily instead. They're already a huge morale boost essentially at no cost.

**Comment by lucianposton** [comment]
Consider moving this to the Degenerate Days Ahead mod.

**Comment by Coolthulhu** [comment]
Note that one use case of them is dilding self to sleep to speed up regeneration.
They're essentially addiction-free sleeping pills, in this PR also free to use.

Also note that heroin is just 50 morale. An hour of vibrating is over 3 lethal heroin overdoses.

And honestly, the whole thing looks like something that would better fit in pouar's fork than a game that pretends to have simulations in it.

**Comment by Tsunder** [comment]
the use case of intentionally exhausting the player is only caused by there being no other free and easy ways to do so. I think this that the fatigue/sleep issue is greater than just this item.  

I think lowering exhaustion would be in the general case a buff to the item

I have no benchmarks for comparing effectiveness of vibrators against heroin. lowered morale cap to 40 (originally 320)

**Comment by kevingranade** [comment]
Removing the only resource cost for using the item is a non-starter unless the benefits are nerfed into oblivion (for example by making the morale bonus decay as fast as it's added so there's no net benefit, and merging the tiredness to where it's simply slightly more effective than waiting) and as has been pointed out, higher exhaustion is a plus for the most common use case, so your nerf is actually a buff.

I also agree in broad strokes (heh) that the vibrator as it is right now is highly exaggerated, it could stand to lose some morale intensity and duration, as well as a lowered exhaustion effect.

Ideally we'd have seperate "fatigue" and "tiredness", and this would only affect tiredness, but even then, solid needs to the overall efficacy are called for.

**Comment by Theundyingcode** [comment]
> This misses the point that there should not be any "free and easy" ways to induce sleep, it's a real problem in some circumstances, and it requires resource expenditures to deal with it.

I think it would still be useful to somehow add the ability to "do 100 burpees" or something to tire yourself out quickly.  Self play with or without a working toy should induce tiredness.  I think the question needing answering is what is a good "cost" for purposely inducing tiredness.  Stamina going bye bye for anything physical should be a given and it should take significantly longer to recover but that shouldn't really effect most people though

Currently, I use low-grade methamphetamine (it's come down hits hard) to make me tired quickly or cough syrup to over come insomnia.  These can be used as the two extremes for balancing the costs of inducing tiredness.

**Comment by Tsunder** [comment]
fatigue is a resource cost as are time, but it seems like at this point the tiredness mechanic is more of a scale than a resource. 

i do agree that sleep shouldn't be easy to induce, but it's neither realistic nor good that not inducing sleep means to not have tiring activities for the player to choose to do. 

as such, the fatigue reduce to 2 (overal decrease of 2 fatigue/min from current, or 50%)

hopefully this means fewer players using vibrators to put themselves to sleep.

**Comment by Coolthulhu** [comment]
Stamina only matters in combat. By adding a stamina cost, you suggest that it will be used in those contexts.

**Comment by Tsunder** [comment]
it makes the player notably out of breath/ less effective in combat during/after using the item, which i think is fair

**Comment by Theundyingcode** [comment]
I wasn't trying to suggest that stamina costs should be considered one of the costs.  I was just saying in that such actions should realistically use up stamina as well.  It should only matter in the very rare case of wandering Zeds appearing as your doing it/ trying to sleep.

**Comment by kevingranade** [comment]
tl;dr, drop fatigue to one per tick (that's still 5x the normal fatigue rate) for un-powered use and I'm good to merge this.  I agree with Coolthulu that stamina burn is irrelevant, but I don't care that much either way.

> I think it would still be useful to somehow add the ability to "do 100 burpees" or something to tire yourself out quickly.

> it's neither realistic nor good that not inducing sleep means to not have tiring activities for the player to choose to do.

Look into adding something else to tire the player out then, it has roughly nothing to do with the vibrator.  Presumably focused exercise is going to increase hunger and thirst as well, so there's the cost it needs to be balanced.

While you're at it, add fatigue costs for running and combat. You're proposing changes to fine-tune a system that's barely implemented in the first palce, which pretty much makes it look like you're just fucking around instead of actually trying to make the game better.

The vibrator effects are wildly nonrepresentative of reality, so adjusting them as if they were "it makes sense to use them without batteries too" doesn't hold up.

It's the same as the various drugs, I don't care if weed has some particular effect IRL, because the representaton of it in the game has nothing to do with weed IRL.

**Comment by ituluwituluwzev** [comment]
It is as useless as any random plastic chunk if it is unpowered or unlubricated.

**Comment by Tsunder** [comment]
fair enough, was toying with the stamina cost when mentioned by @Theundyingcode , probably best to leave it to some overarching rework of stamina and actions

the activity now should bring it in line with an active, more tiring, and slightly more effective version of the mp3 player/stereo system. no longer stacks to euphoric numbers nor tires the player out to sleep after a few uses.
